src/panels/config/integrations/integration-panels/zha/zha-add-devices-page.ts

```diff
@@ -175,8 +174,15 @@ class ZHAAddDevicesPage extends LitElement {
         }
       }
     }
-    if (message.type && message.type === "device_fully_initialized") {
-      this._discoveredDevices.push(message.device_info);
+    if (
+      message.type &&
+      [
+        "device_joined",
+        "raw_device_initialized",
+        "device_fully_initialized",
+      ].includes(message.type)
```

Comment:
Can we create a Const above the class for these message types

Reply:
Actually to the zha data file

Reply:
done